### DISCUSSION TITLE
Universal Control Monterey 12.3

### DISCUSSION THREAD

**Discussion by pepocrespo** [daliansky/XiaoMi-Pro#632] [General]
Has anyone tried the functionality of Universal Control with their Ipad and has it worked? In my case, the Universal Control option does not even appear in the screen tab, but it does in settings. I have a 6th generation iPad. It is simple curiosity in case some adjustment had to be made or it cannot work directly.

**Comment by stevezz3** [comment]
Please refer to https://github.com/OpenIntelWireless/itlwm/issues/292 for AWDL issues.
